plugins/Monolog/Handler/FailureLogMessageDetector.php

```diff
@@ -28,7 +28,9 @@ public function __construct($level = Logger::WARNING)
 
     public function handle(array $record)
     {
-        $this->hasEncounteredImportantLog = true;
+        if ($this->isHandling($record)) {
```

Comment:
can't think of a case where it would hurt to have this in there and fixes the bug where isHandling might be ignored